gui/src/components/WorkspaceNav.svelte

```diff
@@ -0,0 +1,82 @@
+<script lang="ts">
+  import * as router from 'svelte-spa-router';
+
+  import Home from './mono/home.svelte';
+  import Layers from './mono/layers.svelte';
+  import Storage from './mono/database.svelte';
+  import Networking from './mono/networking.svelte';
+
+  export let workspaceId: string;
+  export let active: string;
+</script>
+
+<aside>
+  <button
+    on:click={() => {
+      router.push(`#/workspaces/${workspaceId}`);
+    }}
+    class={active === 'Dashboard' ? 'active' : ''}
```

Comment:
There is a nice shorthand for this pattern!

I think it is this:

```
    class:active={active === 'Dashboard'}
```

See: https://svelte.dev/docs#class_name